### PR TITLE
Remove version number from reverse dependencies page subtitle

### PR DESCRIPTION
**Pull request by sonalkr132**
The page doesn't show reserve dependencies for the latest version
for the gem, it is confusion to show latest version in the subtitle.

### CODE REVIEW

File: app/views/reverse_dependencies/index.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <% @title = t('.title', name: @rubygem.name) %>
-<% @subtitle = @latest_version&.slug %>
+<% @subtitle =  t(".subtitle", name: @rubygem.name) %>
```
Comment (colby-swandale) [inline app/views/reverse_dependencies/index.html.erb:2]: Can we add a line break between the title so this just looks nice on laptop screens.

Comment (sonalkr132) [reply app/views/reverse_dependencies/index.html.erb:2]: These variables are part of the layout, so any changes here will be done on all pages.

Comment (colby-swandale) [reply app/views/reverse_dependencies/index.html.erb:2]: 🤦

Comment (sonalkr132) [reply app/views/reverse_dependencies/index.html.erb:2]: I agree with the long subtitle text this looks non-pleasing. Maybe we can a new variable in layout which uses linebreak?
```
if @newline_subtitle
   tag on new line
```

**Comment by sonalkr132** [comment]
@colby-swandale I went ahead with new var for subtitle on a new line.

**Pull request merged**